File: structuraAlternativa/problema15.py
#Sa se verifice daca patru numere a,b,c si d date de la tastatura pot forma o multime.


a = int(input())
b = int(input())
c = int(input())
d = int(input())

if(a != b)and(b != c)and(c != d)and(a != c)and(a != d)and(b != d):
    print("a, b, c si d pot forma o multime")
else:
    print("a, b, c si d nu pot forma o multime")

File: structuraAlternativa/problema15.cpp
#include <iostream>

//Sa se verifice daca patru numere a,b,c si d date de la tastatura pot forma o multime.

int a;
int b;
int c;
int d;

int main() {

    std::cin >> a;
    std::cin >> b;
    std::cin >> c;
    std::cin >> d;

    if(a != b && b != c && c != d && a != c && a != d && b != d) {

        std::cout << "numerele pot forma o multime";

    } else {

        std::cout << "numerele nu pot forma o multime";

    }

}